package.json

```diff
@@ -16,28 +16,30 @@
     "ipfs-pubsub-1on1": "~0.0.4",
     "logplease": "^1.2.14",
     "multihashes": "^0.4.12",
+    "orbit-db-access-controllers": "github:orbitdb/orbit-db-access-controllers",
     "orbit-db-cache": "~0.2.4",
     "orbit-db-counterstore": "github:shamb0t/orbit-db-counterstore#feat/new-ipfs-log",
     "orbit-db-docstore": "~1.4.3",
     "orbit-db-eventstore": "~1.4.0",
     "orbit-db-feedstore": "~1.4.0",
-    "orbit-db-identity-provider": "github:orbitdb/orbit-db-identity-provider",
+    "orbit-db-identity-provider": "github:orbitdb/orbit-db-identity-provider#feat/verify-identity",
     "orbit-db-keystore": "github:orbitdb/orbit-db-keystore#fix/verification",
     "orbit-db-kvstore": "~1.4.0",
     "orbit-db-pubsub": "~0.5.5",
     "orbit-db-store": "github:shamb0t/orbit-db-store#feat/new-ipfs-log"
   },
   "devDependencies": {
+    "@colony/purser-software": "^1.1.2",
```

Comment:
Not needed I believe, moved to https://github.com/orbitdb/orbit-db-access-controllers